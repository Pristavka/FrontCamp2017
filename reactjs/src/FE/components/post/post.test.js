import React from 'react';
import Post from './post';

it('Should render a Post', () => {
  const post = {
    'author': 'Siarhei',
    'text': 'Hello World!'
  }

  const wrapper = shallow(<Post post={post}/>);
  expect(wrapper).toMatchSnapshot();
});

