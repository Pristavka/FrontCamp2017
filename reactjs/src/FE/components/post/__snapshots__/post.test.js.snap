// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Post 1`] = `
<React.Fragment>
  <div
    className="post"
  >
    <h2
      className="post__name"
    >
      Siarhei
    </h2>
    <div
      className="post__description"
    >
      Hello World!
    </div>
  </div>
  <hr />
</React.Fragment>
`;
